docs/index.md

```diff
@@ -36,6 +36,8 @@ resource "honeycombio_marker" "hello" {
 }
 ```
 
+More advanced examples may be found in the [example directory](/example/).
```

Comment:
As this is most commonly viewed on the [Terraform Registry listing](https://registry.terraform.io/providers/honeycombio/honeycombio/latest/docs), I think this will need to be a direct link
```suggestion
More advanced examples can be found in the [example directory](https://github.com/honeycombio/terraform-provider-honeycombio/tree/main/example).
```

Reply:
A fair point, that was my first iteration and then I tried to be clever 🙂